crates/ruff_linter/src/preview.rs

```diff
@@ -125,3 +125,8 @@ pub(crate) const fn is_check_file_level_directives_enabled(settings: &LinterSett
 pub(crate) const fn is_readlines_in_for_fix_safe(settings: &LinterSettings) -> bool {
     settings.preview.is_enabled()
 }
+
+// https://github.com/astral-sh/ruff/pull/17836
+pub(crate) const fn is_implicit_optional_fix_safe(settings: &LinterSettings) -> bool {
```

Comment:
It would be nice if we can keep the `_postfix`. It helps readers to easily recognize this as a preview check